<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vocabulary</class>
 <widget class="QMainWindow" name="Vocabulary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 171, 143);</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="3">
     <spacer name="vertical_spacer3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="push_button_back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTableWidget" name="table_words_widget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(205, 171, 143);</string>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>word</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="background">
        <color>
         <red>181</red>
         <green>131</green>
         <blue>90</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>translate</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="background">
        <color>
         <red>152</red>
         <green>106</green>
         <blue>68</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>progress</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>134</red>
         <green>94</green>
         <blue>60</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="background">
        <color>
         <red>134</red>
         <green>94</green>
         <blue>60</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="vertical_spacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vertical_layout">
      <item>
       <widget class="QToolButton" name="microfone_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>25</width>
          <height>9</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="push_button_zero_out_progres">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 97, 81);</string>
      </property>
      <property name="text">
       <string>Zero out progress</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="push_button_save_word">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 131, 90);</string>
      </property>
      <property name="text">
       <string>Save to vocab</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <layout class="QGridLayout" name="grid_layout1">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item row="2" column="0">
       <spacer name="vertical_spacer_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>58</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <widget class="QLCDNumber" name="lcd_words_count">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="profile_open_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_word_size">
        <property name="text">
         <string>Current vocabulary size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <spacer name="horizontal_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="push_button_delete_word">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(241, 192, 156, 203);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
